geoh5py/shared/utils.py

```diff
@@ -864,3 +865,26 @@ def remove_duplicates_in_list(input_list: list) -> list:
     :return: The sorted list
     """
     return sorted(set(input_list), key=input_list.index)
+
+
+def resilient_copy(source: str | Path, destination: str | Path):
```

Comment:
**issue:** desired behaviour is not to copy systematically

We want a "resiliant move". The code should still do a move when it can.

The suggestion was telling about a `resiliant_copy` function, but only to pass it a fallback to `shutils.move`:
```py
shutils.move(source, destination, resiliant_copy)
```

Now, there are only a few places to change: no need for anything complicated, no need for a new function: simply replace the few calls to `shutils.move(...)` by `shutils.move(..., copy)`

This tells `shutils.move()` to use `copy` as a fallback instead of `copy2`.
Since we "move" from a tmp folder, or for a save_as, we do not care about preserving file attributes though `copy2`